srcpkgs/font-libertine-otf/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'font-libertine-otf'
+pkgname=font-libertine-otf
+version=5.3.0
+revision=1
+_date=2012_07_02
+archs=noarch
+depends="font-util xbps-triggers"
+font_dirs="/usr/share/fonts/OTF"
+create_wrksrc=yes
+short_desc="Libertine Fonts - Libre multilingual font family"
+maintainer="Pascal Vanier <pascal.vanier@gmail.com>"
+license="OFL-1.1"
+homepage="http://libertine-fonts.org/"
+distfiles="${SOURCEFORGE_SITE}/project/linuxlibertine/linuxlibertine/${version}/LinLibertineOTF_${version}_${_date}.tgz"
```

Comment:
why not add the distfile of font-libertine-ttf here and create a subpackage for it ?

Reply:
Won't that make people download both packages when they only want one ?